examples/default/ios/InstabugTests/InstabugAPMTests.m

```diff
@@ -127,24 +127,53 @@ - (void) testEndExecutionTrace {
 
   OCMStub([apmMock startExecutionTraceWithName:traceName]).andReturn(trace);
   [self.instabugBridge startExecutionTrace:traceName :traceId :resolve :reject];
-  
+
   OCMStub([traceMock end]);
   [self.instabugBridge endExecutionTrace:traceId];
   OCMVerify([traceMock end]);
 }
 
+- (void) testStartFlow {
+  id mock = OCMClassMock([IBGAPM class]);
+  NSString* appFlowName = @"APP_Flow_1";
+
+  OCMStub([mock startFlowWithName:appFlowName]);
```

Comment:
I believe there's no need for `OCMStub` here since we already use `OCMClassMock` which doesn't call the original implementation for the underlying methods on the class and we aren't providing a custom stub here.

Same for the other usages of `OCMStub` in the PR.